spec/definitions/DashboardResponse.yaml

```diff
@@ -0,0 +1,55 @@
+type: object
+properties:
+  metrics:
+    type: object
+    properties:
+      metricName:
+        type: object
+        description: The metric name which was specified in the request with its data.
+        properties:
```

Comment:
Seems incorrect to me.  I don't see why there is a separate section to "segments", as there should only be segments.  Also, it seems there are other properties missing that should be there, like the metric name itself.

Reply:
> seems there are other properties missing that should be there, like the metric name itself

The `metricName` is specified as a key for the metric settings. I didn't know how to say that "in swagger", so I explained this in description.

> I don't see why there is a separate section to "segments", as there should only be segments

I wasn't sure about the task, so I made configurable metrics (the client can choose what to show) and dependent segments.